bin/ember-template-lint.js

```diff
@@ -45,9 +45,10 @@ function lintSource(linter, options, shouldFix) {
 
 function expandFileGlobs(filePatterns, ignorePattern) {
   let result = new Set();
+  let supportedExtensions = new Set(['.hbs', '.html']);
 
   filePatterns.forEach((pattern) => {
-    let isHBS = pattern.slice(-4) === '.hbs';
+    let isHBS = path.extname(pattern) === '.hbs';
```

Comment:
wouldn't we need to check for `.html` here too?

Reply:
good catch! changes landed